### PR TITLE
Update styling of what's new page

### PR DESCRIPTION
**Pull request by mtaylorgds**
## What

Update the what's new page to be closer in appearance to the one on Whitehall.

- Structured the page content into a two-thirds column div.
- Updated the styling of the "Last updated" date.
- Put a horizontal line between sections.
- "Tagged" the change as an "improvement".

## Why

These changes are part of the same [story for the what's new page](https://trello.com/c/WabvCEXa) as [the PR that initially created the page](https://github.com/alphagov/manuals-publisher/pull/2103), the intention being to get the basic page live quicker and then iterate on it. That PR did the bulk of the work of introducing the page and pulling in the GDS design system, this PR updates the page's look to more closely align with the what's new page in Whitehall.

## Visuals

Before (integration):
manuals-publisher integration publishing service gov uk_whats-new

After (development):
manuals-publisher dev gov uk_whats-new

## Trello

[Trello story](https://trello.com/c/WabvCEXa).

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Review by kevindew — COMMENTED** [review]
This looks good, nice work - though as said on Slack it'd be good for the PR to include the context of why we're iterating this as it focuses on the what but not the motivation to iterate this design.

There is some information I should share about the approach you've taken to styling this page, which I can see is copied from Whitehall - just so you have more context.

The approach that has been taken to applying CSS to this page is something of an conventional one for GOV.UK apps as it is applying CSS rules to an individual view (the prefix of `app-view-`) which is tended to be only used as a get out of jail card when we're not able to use individual components - we prefer that when we get a new styling need to instead create components, as these fall out of consistency quite easily. 

To get a bit of a feel for how we prefer to organise CSS you might find it useful to read the CSS of the Content Publisher app: https://github.com/alphagov/content-publisher/blob/main/app/assets/stylesheets/application.scss or I'm happy to chat

File: app/views/layouts/design_system.html.erb [hunk 1]
```diff
@@ -17,7 +17,11 @@
 
   <div class="govuk-width-container">
     <main class="govuk-main-wrapper" id="main-content" role="main">
-      <%= yield %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= yield %>
+        </div>
```
[the remark this answers is not in the document]
Comment (mtaylorgds) [reply app/views/layouts/design_system.html.erb:23]: I've updated the PR description to provide more context.

File: app/views/whats_new/index.html.erb [hunk 2]
```diff
@@ -2,51 +2,55 @@
...
+    <li>numbered lists</li>
+    <li>links</li>
+    <li>email links</li>
+  </ul>
 
-<p class="govuk-body">Other formatting, such as tables, will be removed and only plain text pasted. You’ll need to write the Markdown for
-  these separately.</p>
+  <p class="govuk-body">Other formatting, such as tables, will be removed and only plain text pasted. You’ll need to write the Markdown for
+    these separately.</p>
+</section>
```
[the remark this answers is not in the document]
Comment (mtaylorgds) [reply app/views/whats_new/index.html.erb:56]: I thought Rubocop would have caught that 🤷 
Have enabled what I _think_ is the right setting in Rubymine now.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -5,3 +5,7 @@
 @import "govuk_publishing_components/components/layout-header";
 @import "govuk_publishing_components/components/skip-link";
 @import "govuk_publishing_components/components/title";
+
+.app-view-whats-new__last-updated {
+  color: $govuk-secondary-text-colour;
+}
```
[the remark this answers is not in the document]
Comment (mtaylorgds) [reply app/assets/stylesheets/application.scss:11]: Just to be clear on this, we should create a file in the `views/` directory, not in the `views/whats_new/` directory (and not in the `stylesheets/` directory)?

[the remark this answers is not in the document]
Comment (mtaylorgds) [reply app/assets/stylesheets/application.scss:11]: I think I misunderstood and you meant create a `views/` subdirectory within `stylesheets/`?

[the remark this answers is not in the document]
Comment (mtaylorgds) [reply app/assets/stylesheets/application.scss]: Have now made this change.

**Pull request merged**